src/ai/backend/manager/models/image.py

```diff
@@ -410,6 +418,7 @@ def __init__(
         self.labels = labels
         self.resources = resources
         self.status = status
+        self.created_at = created_at
```

Comment:
Do you need to get created_at?

Reply:
In the test code, we should set `created_at` for `ImageData` comparisons to pass without overriding the `__eq__` method.

It's also fine to assign it manually without adding `created_at` to the constructor.